conda/recipes/cugraph/meta.yaml

```diff
@@ -71,6 +71,7 @@ requirements:
     {% else %}
     - cuda-python >=12.0,<13.0a0
     {% endif %}
+    - aiohttp
```

Comment:
I'd put this above `cuda-version`, just below `run:`.

Reply:
Oh, Sorry about that.